pkg/smartcontract/contract.go

```diff
@@ -34,3 +35,15 @@ func CreateMultiSigRedeemScript(m int, publicKeys keys.PublicKeys) ([]byte, erro
 
 	return buf.Bytes(), nil
 }
+
+// CreateSignatureRedeemScript creates a script runnable by VM.
+func CreateSignatureRedeemScript(publicKey *keys.PublicKey) ([]byte, error) {
```

Comment:
Is it used anywhere?

Reply:
It was, but then I removed this code, fixed.